apps/map_editor/src/model.rs

```diff
@@ -145,26 +145,6 @@ impl Model {
 
         self.recreate_world(ctx, &mut Timer::throwaway());
     }
-
-    fn compute_bounds(&self) -> Bounds {
```

Comment:
It's very nice to not need to figure out `Bounds` upfront for `World` and `FindClosest`